synapse/storage/databases/main/event_push_actions.py

```diff
@@ -1016,7 +1016,7 @@ def _rotate_notifs_before_txn(
         Args:
             txn: The database transaction.
             old_rotate_stream_ordering: The previous maximum event stream ordering.
-            rotate_to_stream_ordering: The new maximum event stream ordering to summarise.
+            rotate_to_stream_ordering: The new maximum event stream ordering to summarize.
```

Comment:
"summarise" is accepted in British English, fyi :)

Reply:
I was matching what was used in the rest of the module (which seems to use `summarized`)!

Reply:
I made this all consistent to British English.

Reply:
sorry, that wasn't meant to be a demand that you change it back! I'd assumed you were correcting this because you thought it was an isolated typo.

anyway, consistency ftw, thanks!